src/common/snippets/src/generator.cpp

```diff
@@ -104,6 +104,8 @@ RegType Generator::get_op_out_reg_type(const ov::Output<Node>& out) const {
              std::dynamic_pointer_cast<op::HorizonSum>(op) ||
              std::dynamic_pointer_cast<op::Fill>(op))
         return RegType::vec;
+    else if (std::dynamic_pointer_cast<ov::op::v1::Reshape>(op))
```

Comment:
Probably I missed something but as I can see, there should be `ov::snippets::op::Reshape`, shouldn't it?

Reply:
done, thanks Alexandra!